bzip2.rs

```diff
@@ -975,19 +977,25 @@ unsafe fn saveInputFileMetaInfo(srcName: *mut c_char) {
         ioError();
     }
 }
-#[cfg(unix)]
-unsafe fn applySavedTimeInfoToOutputFile(dstName: *mut c_char) {
-    let mut uTimBuf: utimbuf = utimbuf {
-        actime: 0,
-        modtime: 0,
-    };
-    uTimBuf.actime = fileMetaInfo.st_atime;
-    uTimBuf.modtime = fileMetaInfo.st_mtime;
-    let retVal = libc::utime(dstName, &uTimBuf);
-    if retVal != 0 as libc::c_int {
-        ioError();
+
+fn apply_saved_time_info_to_output_file(dst_name: &CStr, metadata: Metadata) {
+    #[cfg(unix)]
```

Comment:
This will cause a warning about `dst_name` and `metadata` being unused when compiling for a non-unix target.